service/user/service.go

```diff
@@ -0,0 +1,112 @@
+package user
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+	"github.com/stasBigunenko/monorepa/pkg/storage/newStorage"
+)
+
+type UsrService struct {
+	Us      model.UserHTTP
```

Comment:
Why we need this?

Reply:
we return model.UserHHTP,in most of function, i thought would be easily with return

Reply:
done